### ISSUE TITLE
About the "Install the CA certificate"

### ISSUE THREAD

**Issue by 3gstudent** [stufus/certerator#1]
In your blog : 
https://labs.mwrinfosecurity.com/blog/masquerading-as-a-windows-system-binary-using-digital-signatures/.

![11111](https://cloud.githubusercontent.com/assets/15814408/14697061/7de4a18c-0735-11e6-82a1-bfc7e96d8749.png)

I think we need the file "ca.crt" to Install the CA certificate,but the certerator.py did not create the file.
I'm confused whether or not we should change the certerator.py.
Could you give me some advice?
Many thanks.

3gstudent
@3gstudent

**Comment by stufus** [comment]
Good point, I renamed the file in the code. It would be ca.pem, ca.cer etc depending on the configured filename.

**Comment by 3gstudent** [comment]
Thanks.Would you change the code or I rename the ca.pem, ca.cer etc?

**Comment by ghost** [comment]
certerator#1
